benefits/templates/admin/base_site.html

```diff
@@ -1,5 +0,0 @@
-{% extends "admin/base_site.html" %}
-
-{% block extrahead %}
-  {% include "admin/includes/favicon.html" %}
-{% endblock extrahead %}
```

Comment:
I decided to remove `base_site` as it is not necessary to override, and it's hard to keep track of multiple places of things being overwritten.